scripts/createImage.sh

```diff
@@ -38,11 +39,16 @@ if [[ "$SPARK_VERSION" == "3."* ]] && ( [[ "$SCALA_BIN_VERSION" == "2.13" ]] ||
     ./build/mvn --batch-mode clean
     ./build/mvn --batch-mode package -pl examples
     ./build/mvn --batch-mode package -Pkubernetes -Pscala-$SCALA_BIN_VERSION -pl assembly
-    ./bin/docker-image-tool.sh -t "$image_tag" build
+    # including python in spark image
+    ./bin/docker-image-tool.sh -t "$image_tag" -p ./resource-managers/kubernetes/docker/src/main/dockerfiles/spark/bindings/python/Dockerfile build
```

Comment:
This always builds the base image with Python. The `-p ...` argument should be used conditionally.